bazaar/interface.rst

```diff
@@ -1,12 +1,207 @@
 
-Bazaar Interfaces
-*****************
+Edustore Distributed API
+=========================
 
-LMS
-===
+draft v0.1
 
+.. contents::
 
-CMS
-===
 
+Signing and Authentication 
+==========================
 
+There are two supported methods:
+
+1. Header HMAC256 AUTH token in headers and JSON body (Similar: http://docs.aws.amazon.com/AmazonS3/latest/dev/RESTAuthentication.html)
+2. GET/POST request with params=<json>&authentication=<client_id:hash>
```

Comment:
More than one line description of the usage of parameters would be nice. This is now more than confusing to the reader what this means. When is GET and POST used? Explanation what is "params" and "authentication". What does "client_id:hash" mean? Is it one value or two? Should it be "<client_id>:<hash>"?